### ISSUE TITLE
TypeError: Cannot read property 'test' of undefined

### ISSUE THREAD

**Issue by jedmao** [postcss/postcss-bem-linter#65]
I'm using postcss-bem-linter in my webpack configuration and everything is working find, defining blocks and all, until I define utilities:

```css
/** @define utilities */

.MicrosoftMap .MapPushpinBase {
    cursor: pointer !important;
}
```

```
ERROR in ./~/css-loader!./~/postcss-loader!./styles/sheets/store-locator/index.css
Module build failed: TypeError: Cannot read property 'test' of undefined
    at Z:\project\node_modules\postcss-bem-linter\lib\validate-utilities.js:22:19
    at Array.every (native)
    at isValid (Z:\project\node_modules\postcss-bem-linter\lib\validate-utilities.js:21:20)
    at Z:\project\node_modules\postcss-bem-linter\lib\validate-utilities.js:11:9
    at Array.forEach (native)
    at module.exports (Z:\project\node_modules\postcss-bem-linter\lib\validate-utilities.js:10:18)
    at checkRule (Z:\project\node_modules\postcss-bem-linter\index.js:46:9)
    at Z:\project\node_modules\postcss-bem-linter\index.js:40:9
    at Array.forEach (native)
    at Z:\project\node_modules\postcss-bem-linter\index.js:37:14
```

**Comment by davidtheclark** [comment]
Can you please share your configuration?

**Comment by jedmao** [comment]
```js
require('postcss-bem-linter')({ 
	componentName: new RegExp(`^${WORD}$`), 
	componentSelectors(block) { 
		const element = `(?:__${WORD})?`; 
		const modifier = `(?:--${WORD})?`; 
		return new RegExp(`^\\.${block}${element}${modifier}$`); 
	} 
}) 
```

The webpack config is:

```js
  module: {
    loaders: [
      { test: /\.css$/, loader: 'style-loader!css-loader!postcss-loader' }
    ]
  },
```

**Comment by davidtheclark** [comment]
Thanks. Looks like it's expecting you to have a utilitySelectors() function but you haven't defined one. Maybe I should provide a more meaningful error, like "You use @define utilities but haven't provided a convention for utility classes." Would that be a good solution?

**Comment by jedmao** [comment]
That would be very helpful!

**Comment by davidtheclark** [comment]
@jedmao : I'll put this out in a release real soon, once I've also handled #63.

**Comment by jedmao** [comment]
:+1:

**Comment by davidtheclark** [comment]
Released in 1.1.0
